firmware/console/binary/tunerstudio.cpp

```diff
@@ -743,6 +766,10 @@ int tunerStudioHandleCrcCommand(ts_channel_s *tsChannel, char *data, int incomin
 	} else if (command == TS_PAGE_COMMAND) {
 		uint16_t page = *(uint16_t *) data;
 		handlePageSelectCommand(tsChannel, TS_CRC, page);
+	} else if (command == TS_GET_STRUCT) {
```

Comment:
Could this use the `getOutputChannels` or reading pages? Do we need to duplicate this, or can the 'normal' output channels just be one case for GET_STRUCT?

Reply:
there is actually a code comment somewhere along these lines.

I am waiting on Tuner Studio to fix https://github.com/rusefi/rusefi/issues/794 - once they fix there is a chance I will make _all_ these data structures part of dynamically generated virtual getOutputChannels  mega-page. This case we would get logging of any of those if needed

for now it's easier to copy-paste a bit to keep code simpler.